seaborn/utils.py

```diff
@@ -527,3 +527,29 @@ def categorical_order(values, order=None):
                     order = order
         order = filter(pd.notnull, order)
     return list(order)
+
+
+def to_unicode_or_bust(obj, encoding='utf-8'):
```

Comment:
Actually this is a "classic" function which I copied (the docstring credits the source). I didn't think much about the name.

But it will fail if the object includes invalid characters according to the specified encoding.